Tests/GRPCInProcessTransportTests/InProcessClientTransportTests.swift

```diff
@@ -171,7 +171,7 @@ final class InProcessClientTransportTests: XCTestCase {
       }
 
       group.addTask {
-        try await server.listen { stream in
```

Comment:
Aren't we getting a warning because we're not using `context` here? Should we `_` it, or should we assert the contained method descriptor?

Reply:
Nope, no warning for not using it. I don't mind whether we change it to `_` or leave it as `context`. Let me know what you prefer.

I'll add a separate test checking we get the descriptor in the context.

Reply:
Ah, that's fine, I'm okay with the name. I thought warnings were emitted in this case :D